examples/mixture_density_network.py

```diff
@@ -5,7 +5,8 @@
 Probability model:
     Prior: None ("flat prior")
     Likelihood: Mixture sum of normals parameterized by a NN
-    Inference: Maximum a posteriori
+    
```

Comment:
can i also say that there shouldn't be an empty line between probability model and inference/variational model? :) (we could, although for consistency we would need to do it for all the examples)